app/styles/currency-input.less

```diff
@@ -47,6 +47,37 @@
       outline: none;
     }
 
+    &:focus-within {
+      border: 1px solid var(--color-border-default);
+    }
+  }
+  .currency-input-onlycurrency {
+    padding-right: 0px;
+    .currency-selector {
+      height: 100%;
+      display: flex;
+      align-items: center;
+      padding-right: var(--spacing-px-12);
+      margin-right: 0px;
+      width: 100%;
+    }
```

Comment:
There are only a few css attributes that are different from the base `.currency-input` : width, margin, border...
Instead of duplicating the whole section, define a class that only overrides the attributes you need.

In your template you should do something like :
```html
 <div class="currency-input {{if this.onlyCurrencyInput "only-selector"}} ..." ...>
```
And then in the css, override the currency-input attributes that are different in `.only-selector`